### PR TITLE
Caption this arun jeyaraman

### PR DESCRIPTION
**Pull request by ArunJeyaraman**

### CODE REVIEW

**Comment by johnannamalai** [comment]
Approved

**Review by johnannamalai — APPROVED** [review]

**Review by PushparaniChellakani — APPROVED** [review]

File: _slides/03-ArunJeyaraman.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+
+Blue Octo
```
Comment (chrisn555-im) [inline _slides/03-ArunJeyaraman.md:7]: I really like this change

**Review by kachirem — APPROVED** [review]
approved

**Review by chrisn555-im — APPROVED** [review]
Looks good

**Pull request merged**